src/slide-potentiometer-element.ts

```diff
@@ -6,14 +6,17 @@ export class SlidePotentiometerElement extends LitElement {
   @property() value = 0;
   @property() minValue = 0;
   @property() maxValue = 50;
+  @property() mouseX = 0;
+  @property() mouseY = 0;
+  @property() rotation = 0;
```

Comment:
those three (mouseX/mouseY/rotation) seem unused?

Reply:
Yes, I had a bunch of extra values displayed when trying to make it work. Forgot to remove these properties will do.